packages/svelte/src/compiler/phases/3-transform/client/visitors/template.js

```diff
@@ -2193,11 +2191,7 @@ export const template_visitors = {
 					return context.state.analysis.runes ? assign : b.sequence([assign, invalidate]);
 				} else {
 					const original_left = /** @type {import('estree').MemberExpression} */ (assignment.left);
-					const left = b.member(
-						expression_for_other,
-						context.visit(original_left).property,
-						original_left.computed
-					);
```

Comment:
I honestly can't remember anymore why I originally did it like this, but if the tests pass I guess all is good 😄